src/style.css

```diff
@@ -0,0 +1,31 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  
+}
+
+.stars__star:not(:first-child) {
+  margin-left: 4px;
+}
+
+.stars__gold {
```

Comment:
Interesting solution 👍
But in this task we learn how to you use pseudo-classes in CSS, so please try to redo it using them.
It would be useful for you
https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes#tree-structural_pseudo-classes